plugins/source.go

```diff
@@ -114,19 +116,21 @@ func (p *SourcePlugin) Sync(ctx context.Context, logger zerolog.Logger, spec spe
 	if concurrency == 0 {
 		concurrency = defaultConcurrency
 	}
-	logger.Info().Uint64("concurrency", concurrency).Msg("starting fetch")
+	logger.Info().Uint64("concurrency", concurrency).Msg("starting sync")
```

Comment:
nit: maybe starting `source sync` just to distinguish between CLI sync (which is source->destination) or should we keep it fetch...naming is hard....